src/combinators/compose2.spec.js

```diff
@@ -0,0 +1,59 @@
+const test = require('tape')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+
+const isFunction = require('../core/isFunction')
+
+const compose2 = require('./compose2')
+
+test('compose2', t => {
+  const fn = bindFunc(compose2)
+  const f = x => y => x * y
+  const g = x => x - 1
+  const h = x => x + 1
+  const x = 22
+  const y = 1
+
+  t.ok(isFunction(compose2), 'is a function')
+
+  const err = /TypeError: compose2: First, second and third arguments must be functions/
```

Comment:
Interesting, including `TypeError`. Thinking we should do this for all cases.
should this include the `^` at the start, JIC.
```js
const err = /^TypeError: compose2: First, second and third arguments must be functions/
```

Reply:
It doesn't hurt to add the ^ and i think being as specific as possible without being limiting is the best call